modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc

```diff
@@ -225,7 +225,7 @@ class NDB_Menu_Filter_Form_Acknowledgements extends NDB_Menu_Filter_Form
         return array_merge(
             $baseDeps,
             array(
-             $baseurl . '/acknowledgements/js/acknowledgements.js',
+             $baseurl . '/acknowledgements/js/acknowledgements_helper.js',
```

Comment:
Not that it makes much difference, but is there any reason for all the js file renames?